src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileUpdator.java

```diff
@@ -148,7 +150,8 @@ private HttpStatus getHttpStatusFromFeignException(FeignException ex) {
     private UserProfile validateUserStatus(String userId) {
         Optional<UserProfile> userProfileOptional = userProfileRepository.findByIdamId(userId);
         if (!userProfileOptional.isPresent()) {
-            throw new ResourceNotFoundException("could not find user profile for userId: or status is not active " + userId);
+            throw new ResourceNotFoundException("could not find user profile for userId: or status is not active "
```

Comment:
only issue I can see here if userId value null we can get null pointer...hence are we sure we are getting userId in all cases? with + null pointer we can't get. but with concat we my get null pointer.